Add (more) explicit references to Jinja2 docs in conditionals doc (#61814)
ansible/ansible: docs/docsite/rst/user_guide/playbooks_conditionals.rst

        when: ansible_facts['os_family'] == "Debian"
        # note that all variables can be used directly in conditionals without double curly braces

You can also use `parentheses to group and logical operators <https://jinja.palletsprojects.com/en/master/templates/#logic>`_ to combine conditions::

    tasks:
      - name: "shut down CentOS 6 and Debian 7 systems"
        command: /sbin/shutdown -t now
        when: (ansible_facts['distribution'] == "CentOS" and ansible_facts['distribution_major_version'] == "6") or
              (ansible_facts['distribution'] == "Debian" and ansible_facts['distribution_major_version'] == "7")

> Multiple conditions that all need to be true (i.e., a logical ``and``) can also be specified as a list::

Reviewer: We working to remove uses of `i.e.` from the docs (#62224)
```suggestion
Multiple conditions that all need to be true (such as a logical ``and``) can also be specified as a list::
```
